.idea/misc.xml

```diff
@@ -6,4 +6,4 @@
   <component name="ProjectType">
     <option name="id" value="Android" />
   </component>
-</project>
+</project>
```

Comment:
Can you revert this? Let's keep EOF line endings for consistency.

Reply:
its resolved